assets/js/atomic/blocks/product/title/style.scss

```diff
@@ -22,4 +22,15 @@
 			width: 10em;
 		}
 	}
+	/*rtl:begin:ignore*/
+	.wc-block-components-product-title__align-left {
```

Comment:
Do we control this, because I thought modifiers used `--` not `__`

Reply:
Fixed.